src/videojs-http-streaming.js

```diff
@@ -597,6 +597,7 @@ class VhsHandler extends Component {
     this.options_.customTagParsers = this.options_.customTagParsers || [];
     this.options_.customTagMappers = this.options_.customTagMappers || [];
     this.options_.cacheEncryptionKeys = this.options_.cacheEncryptionKeys || false;
+    this.options_.llhls = Boolean(this.options_.llhls);
```

Comment:
I could be wrong, but this would still default to `false`, right?

Reply:
Yes

Reply:
I think we want it to default to `true`, though. Maybe something like this?
```suggestion
    this.options_.llhls = this.options_.hasOwnProperty('llhls') ? this.options_.llhls : true;
```